src/include/gurt/common.h

```diff
@@ -94,12 +94,15 @@ extern "C" {
 		if ((cond) && (ptr) != NULL) {				\
 			if (count <= 1)					\
 				D_DEBUG(DB_MEM,				\
-					"alloc(" #func ") '" name "': %i at %p.\n", \
+					"alloc(" #func ") '" name "':   \
+					                  %i at %p.\n", \
```

Comment:
(style)  code indent should use tabs where possible
(style)  Avoid unnecessary line continuations
(style)  Avoid line continuations in quoted strings